### PR TITLE
OCM-9896 | ci: Automated id:75233

### PR DESCRIPTION
**Pull request by jerichokeyne**
**What this PR does / why we need it**: Automates another test case: OCP-75233

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-9896](https://issues.redhat.com//browse/OCM-9896)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

File: tests/e2e/trusted_ips_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	// nolint
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+)
+
+var _ = Describe("List trusted IPs", func() {
```
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:13]:
```suggestion
var _ = Describe("Trusted IPs", func() {
```

File: tests/e2e/trusted_ips_test.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		err               error
+		trustedIPsService exec.TrustedIPsService
+	)
+
+	BeforeEach(func() {
+		trustedIPsService, err = exec.NewTrustedIPsService(constants.TrustedIPsDir)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("list trusted IPs - [id:75233]", ci.Day1, ci.Medium, func() {
```
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:26]:
```suggestion
	It("can list trusted IPs - [id:75233]", ci.Day1, ci.Medium, func() {
```

Comment (radtriste) [reply tests/e2e/trusted_ips_test.go:26]: btw, that should be a day1-supplemental or day2

File: tests/e2e/trusted_ips_test.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		By("Get list of trusted IPs")
+		args := exec.TrustedIPsArgs{}
+		err = trustedIPsService.Init()
+		Expect(err).ToNot(HaveOccurred())
```
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:30]: Init is already done when calling `exec.NewTrustedIPsService`, no need to redo it

```suggestion
		err = trustedIPsService.Init()
		Expect(err).ToNot(HaveOccurred())
```

**Comment by jerichokeyne** [comment]
@radtriste Can you take another look when you get a chance?

**Review by radtriste — APPROVED** [review]
/lgtm

**Comment by jerichokeyne** [comment]
@radtriste can you /ok-to-test this PR?

**Comment by jerichokeyne** [comment]
@radtriste do you know why this can't be merged yet? Konflux is saying `Red Hat Konflux is waiting for approval`

**Pull request merged**